integration/apps/rspec/spec/spec_helper.rb

```diff
@@ -1,9 +1,12 @@
 require 'datadog/ci'
 
 # Enable CI tracing
-Datadog.configure do |c|
+Datadog::Tracing.configure do |c|
+  c.instrument :rspec
+end
+
+Datadog::CI.configure do |c|
   c.ci_mode.enabled = true
-  c.use :rspec
 end
```

Comment:
Wait -- is this correct? Shouldn't rspec be automatically instrumented when using CI?

Reply:
I think it's correct; I don't think it auto-activates instrumentation. We may want to allow CI to send `instrument` calls through its own proxy, but I don't see this as important to change either.

Reply:
I guess it accurately reflects what's in the docs: https://docs.datadoghq.com/continuous_integration/setup_tests/ruby/?tab=rspec . I expected it to be automatic, but I guess it's currently now.